plasma_framework/test/src/exits/payment/PaymentExitGame.test.js

```diff
@@ -48,6 +48,8 @@ contract('PaymentExitGame - End to End Tests', ([_, richFather, bob]) => {
     const DEPOSIT_VALUE = 1000000;
     const INITIAL_IMMUNE_VAULTS = 2; // ETH and ERC20 vault
     const INITIAL_IMMUNE_EXIT_GAMES = 1; // 1 for PaymentExitGame
+    const ETH_VAULT_ID = 1;
+    const ERC20_VAULT_ID = 2;
```

Comment:
minor: consider putting these two into `constants.js` (For all the other `VAULT_ID` in other tests too)

Reply:
done